business-registry-dashboard/app/utils/affiliations.ts

```diff
@@ -155,7 +155,14 @@ export const isDissolution = (item: Business) => {
 export const getDetails = (item: Business): EntityAlertTypes[] => {
   const details = []
   if (isExpired(item)) {
-    details.push(EntityAlertTypes.EXPIRED)
+    const typeMap = {
+      [CorpTypes.REGISTRATION]: 'registration',
+      [CorpTypes.INCORPORATION_APPLICATION]: 'incorporation application',
+      [CorpTypes.AMALGAMATION_APPLICATION]: 'amalgamation application',
+      [CorpTypes.CONTINUATION_IN]: 'continuation application'
```

Comment:
What about other languages?

Reply:
Thanks for catching this. Should be fixed now.